ssl/statem/statem_dtls.c

```diff
@@ -1022,7 +1022,7 @@ WORK_STATE dtls_wait_for_dry(SSL *s)
 int dtls1_read_failed(SSL *s, int code)
 {
     if (code > 0) {
-        fprintf(stderr, "invalid state reached %s:%d", __FILE__, __LINE__);
+        fprintf(stderr, "dtls1_read_failed(); invalid state reached\n");
```

Comment:
This is changing the code - personally I don't see the value in the use of __FILE__ and __LINE__ here either but it is a surprise to see a different behaviour given the description.